backend/tracim_backend/lib/utils/request.py

```diff
@@ -186,7 +186,9 @@ def _get_content(self, content_id_fetcher: typing.Callable[[], int]) -> Content:
             content_type=content_type_list.Any_SLUG,
         )
 
-    def _get_content_type(self, content_type_slug_fetcher: typing.Callable[[], str]) -> ContentType:
+    def _get_content_type(
+        self, content_type_slug_fetcher: typing.Callable[[], str]
```

Comment:
with a signature like this, this is *required* to add a docstring explaining what is expected as parameters, as result + give examples

Reply:
not application related, will be fixed in #2626